en/micro-integrator/docs/overview/about-this-release-7.1.0.md

```diff
@@ -0,0 +1,102 @@
+# About this Release
+
+WSO2 Enterprise Integrator (EI) version 7.1.0 is the latest release of the WSO2 EI 7.x family, which provides a powerful solution for integrating systems.
+
+## What's new in this release?
+
+#### Micro Integrator
+
+The following features and improvements are available for the <b>Micro Integrator</b> of EI 7.1.0:
+
+-   [Task coordination](../../setup/deployment/deploying_wso2_ei/#cluster-coordination) for distributed deployments
+-   Capability of viewing and configuring logs without accessing the file system
+-   User management with [external user stores](../../setup/install_and_setup_overview/#data-stores)
+-   [Readiness probe](../../setup/deployment/health_check) for health checking
+-   Enhanced experience for [handling security](../../setup/security/encrypting_plain_text) (Ciphertool, Securevault, Kubernetes Secrets)
+-   [Hot deployment](../../develop/hot-deployment) capability
+-   Schema-based XML to JSON transformations with the [JSON Transform mediator](../../references/mediators/json-Transform-Mediator)
+-   [Swagger generation for data services](../../develop/advanced-development/using-swagger-for-apis)
+-   [Transaction counter](../../setup/deployment/deployment_checklist/#monitoring-transaction-counts) for services
+-   [Environment/System variable support](../../setup/dynamic_server_configurations) for Kubernetes environments
+-   [Enhanced Micro Integrator CLI](../../administer-and-observe/using-the-command-line-interface)
+
+#### WSO2 Integration Studio
+
+The following features and improvements are available for <b>WSO2 Integration Studio</b> EI 7.1.0:
+
+Improvements to [<b>Connector</b>](../../references/connectors/connectors-overview):
```

Comment:
HTML within markdown? Does this formatting render correctly on the site?

Reply:
yeah.. it worked. checked it